src/lib/assets/index.ts

```diff
@@ -53,4 +68,108 @@ export class Assets {
 
     return allYaml(this, rbacMode);
   };
+
+  generateHelmChart = async (basePath: string) => {
+    const CHART_DIR = `${basePath}/${this.config.uuid}-chart`;
+    const CHAR_TEMPLATES_DIR = `${CHART_DIR}/templates`;
+    const valuesPath = resolve(CHART_DIR, `values.yaml`);
+    const chartPath = resolve(CHART_DIR, `Chart.yaml`);
+    const nsPath = resolve(CHAR_TEMPLATES_DIR, `namespace.yaml`);
+    const watcherSVCPath = resolve(CHAR_TEMPLATES_DIR, `watcher-service.yaml`);
+    const admissionSVCPath = resolve(CHAR_TEMPLATES_DIR, `admission-service.yaml`);
+    const mutationWebhookPath = resolve(CHAR_TEMPLATES_DIR, `mutation-webhook.yaml`);
+    const validationWebhookPath = resolve(CHAR_TEMPLATES_DIR, `validation-webhook.yaml`);
+    const admissionDeployPath = resolve(CHAR_TEMPLATES_DIR, `admission-deployment.yaml`);
+    const watcherDeployPath = resolve(CHAR_TEMPLATES_DIR, `watcher-deployment.yaml`);
+    const tlsSecretPath = resolve(CHAR_TEMPLATES_DIR, `tls-secret.yaml`);
+    const apiTokenSecretPath = resolve(CHAR_TEMPLATES_DIR, `api-token-secret.yaml`);
+    const moduleSecretPath = resolve(CHAR_TEMPLATES_DIR, `module-secret.yaml`);
+    const storeRolePath = resolve(CHAR_TEMPLATES_DIR, `store-role.yaml`);
+    const storeRoleBindingPath = resolve(CHAR_TEMPLATES_DIR, `store-role-binding.yaml`);
+    const clusterRolePath = resolve(CHAR_TEMPLATES_DIR, `cluster-role.yaml`);
+    const clusterRoleBindingPath = resolve(CHAR_TEMPLATES_DIR, `cluster-role-binding.yaml`);
+    const serviceAccountPath = resolve(CHAR_TEMPLATES_DIR, `service-account.yaml`);
+
+    // create helm chart
+    try {
+      // create chart dir
+      await createDirectoryIfNotExists(CHART_DIR);
+
+      // create charts dir
+      await createDirectoryIfNotExists(`${CHART_DIR}/charts`);
+
+      // create templates dir
+      await createDirectoryIfNotExists(`${CHAR_TEMPLATES_DIR}`);
+
+      // create values file
+      await overridesFile(this, valuesPath);
+
+      // create the chart.yaml
+      await fs.writeFile(chartPath, chartYaml(this.config.uuid, this.config.description || ""));
```

Comment:
```suggestion
      await fs.writeFile(chartPath, dedent(chartYaml(this.config.uuid, this.config.description || "")));
```
Chart.yaml gen'd with a bunch of leading whitespace when I ran this block -- just looks like a forgotten dedent() call.

Reply:
Or, actually... is there are reason you don't want to "dedent" the chartYaml return string _before_ you return it -- like, move the dendent call inside of chartYaml()?  Same thing with the rest of the methods that use dedent() in this file. :thinking:

1) Seems kinda strange to have the yaml generation functions return content that includes arbitrary whitespace, and 2) if these generation functions are used anywhere else, they're _also_ going to have to be deindented.

Reply:
nope, just an oversight. good catch